todo_app/templates/index.html

```diff
@@ -16,15 +16,20 @@ <h1 class="display-4">To-Do App</h1>
       <input type="submit" value="Submit">
     </div>
   </form>
-
+  
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
         {% for item in items %}
-          <li>{{ item.title }} - {{item.status}}</li>
+        <li>{{ item.id }} - {{ item.name }} - {{item.status}}</li>
+        <form action="/move-to-done/{{item.id}}" method="post" class="form-example">
```

Comment:
Nice. You could also wrap this form in `{% if item.status != 'Done' %} ... {% endif %}`

So that the button doesn't appear for items that are already "done".